src/updateAttribute.js

```diff
@@ -49,14 +49,15 @@ function setAttribute(node, attrName, attrValue, oldAttrValue, isSvgAttribute) {
 
     // get patched event handler
     const patchedHandler = getPatchedEventHandler(node, attrName, attrValue);
+    
+    const isCaptureEvent = attrName.endWith('Capture');
```

Comment:
```suggestion
    const isCaptureEvent = attrName.endsWith('Capture');
```

Reply:
We will also need to add support for onGotPointerCapture, and onLostPointerCapture as well. React does handle it. 
Also, we can have a capture phase on these events, so this becomes more tricky. You can try the below logic in browser.

```js
const para = document.querySelector('p');

para.addEventListener('gotpointercapture', () => {
  console.log('I\'ve been captured!')
}, false);

para.addEventListener('pointerdown', (event) => {
  para.setPointerCapture(event.pointerId);
});

document.body.addEventListener('gotpointercapture', () => {
  console.log('budy I\'ve been captured!')
}, false);

document.body.addEventListener('pointerdown', (event) => {
  para.setPointerCapture(event.pointerId);
});
```
So ideally even this `onGotPointerCaptureCapture` and `onLostPointerCaptureCapture` is valid event.

We will have to add Edge case for this two events.

Reply:
Looks like this recently got fixed in React as well.
https://github.com/facebook/react/blob/master/packages/react-dom/src/events/DOMPluginEventSystem.js#L498

Reply:
Should we handle it the way react does ?
We can also check for these specific events and then do the special logic.

Reply:
We can handle it in the same way. 
https://www.measurethat.net/Benchmarks/Show/9484/0/substr-vs-endswith

We will also have to take care of getEventName method for this